packages/appium/lib/cli/parser.js

```diff
@@ -280,6 +287,38 @@ class ArgParser {
       }
     }
   }
+
+  /**
+   * Add subcommand and sub-sub commands for 'setup' subcommand.
+   * @param {import('argparse').SubParser} subParser
+   */
+    static _addSetupToParser(subParser) {
+      const setupParser = subParser.add_parser('setup', {
+        add_help: true,
+        help: `Install latest drivers '${_.join(DEFAULT_DRIVERS, ',')}' and ` +
+          `plugins '${_.join(DEFAULT_PLUGINS, ',')}' ` +
+          `if APPIUM_HOME has no drivers and plugins`,
+      });
+
+      ArgParser._patchExit(setupParser);
+      const extSubParsers = setupParser.add_subparsers({
+        dest: `setupCommand`,
+      });
+
+      const parserSpecs = [
+        {
+          command: 'all',
+          help: `Install all known drivers '${_.join(_.keys(KNOWN_DRIVERS), ',')}' ` +
```

Comment:
these are not "known", but rather officially supported by Appium team drivers and plugins